src/components/UserGroup.tsx

```diff
@@ -22,18 +22,65 @@ const UserImage = styled(UserPic)`
     display: block;
 `;
 
+const StyledCounter = styled.div`
+    height: 23px;
+    border-radius: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-inline: 7px;
+    background-color: ${gray4};
+    position: relative;
+    right: 10px;
+    top: 1px;
+`;
+const StyledSmallCircle = styled.div`
+    width: 13px;
+    height: 13px;
+    border-radius: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: ${gray6};
+    position: relative;
+    right: 17px;
+    top: 7px;
+`;
+
 export const UserGroup: FC<{
     users: NonNullable<ActivityByIdReturnType>[];
     className?: string;
     size?: number;
 }> = ({ users, className, size = 24 }) => {
+    const showCounter = users.length > 3;
+
     return (
         <UserGroupContainer className={className}>
-            {users.map(({ user, ghost }, i) => (
-                <UserContainer key={i}>
-                    <UserImage src={user?.image} email={user?.email || ghost?.email} size={size} />
-                </UserContainer>
-            ))}
+            {showCounter
+                ? [users[0], users[1], users[2]].map(({ user, ghost }, i) => (
+                      <UserContainer key={i}>
+                          <UserImage src={user?.image} email={user?.email || ghost?.email} size={size} />
+                      </UserContainer>
+                  ))
+                : users.map(({ user, ghost }, i) => (
+                      <UserContainer key={i}>
+                          <UserImage src={user?.image} email={user?.email || ghost?.email} size={size} />
+                      </UserContainer>
+                  ))}
+            {showCounter && (
+                <>
+                    <StyledCounter>
+                        <Text color={gray9} size="s">
+                            {users.length - 3}
+                        </Text>
+                    </StyledCounter>
+                    <StyledSmallCircle>
+                        <Text color={gray9} size="xs">
+                            +
+                        </Text>
+                    </StyledSmallCircle>
```

Comment:
Can it be single element?

Reply:
![image](https://github.com/taskany-inc/issues/assets/9020793/beed9551-452d-4dcc-a448-c0d8857400be)
is this the actual design?

Reply:
no